parachain/primitives/ethereum/src/ethashdata.rs

```diff
@@ -0,0 +1,518 @@
+    hex!("8cb0763f7a0ec5aa0761bfbb049fe2c1"),
+    hex!("ccd344c4e3b6b028253a724d7014b831"),
+    hex!("8a409beab35999899ae65a0b4519e081"),
+    hex!("472da5358a5c51cbe55d4beadfa8d2ef"),
+    hex!("af3eb432bc504607d20dd9b93fdf0382"),
+    hex!("15652ee8ba674184c53238346e7c2818"),
+    hex!("1038ca6c9eca2e2240ae40dcee168d7f"),
+    hex!("8f99e10b2925562e1061f7ad3a7f591e"),
+    hex!("ec7875f6e017ec743ce364f257c79a70"),
+    hex!("df953a9008d4f96c63da1993439a81c6"),
+    hex!("c44353ffa531580a5bbbbb8faa0855cd"),
+    hex!("4d2020a9a71a5351f415552099e2760a"),
+    hex!("39c14bb1c059f585fd4ba723dc1c66ca"),
+    hex!("4fbbf8b5795d2a8b7a83b3768869b5e1"),
+    hex!("8210215bfb4ace609e0fb0973ac511de"),
+    hex!("16fba09f0e8ecd584d0524a4c5723a8b"),
+    hex!("8619f37f7511cec0777ab7efe07ae451"),
+    hex!("a4ff22443285056f6e6be1bba3a886e8"),
+    hex!("a13fb0480ab2fb0cdb3a1373694323f9"),
+    hex!("c9b9c6a82b04c72d8970ec3e015c2607"),
+    hex!("064449044bd883c413684d6d29eef904"),
+    hex!("b1896f87a9ae66ec233de69dd5b5fc50"),
+    hex!("4d39896b61686334603e8accbb0288ff"),
+    hex!("e0687ca2fc99a162a61e83da0f54dfea"),
+    hex!("e12c1c535fc6f6498de80b0da9094c80"),
+    hex!("4731d5e4b9421510c3ddf73d87a9c4a3"),
+    hex!("247a38f6f0fcc658fc35057787bd9054"),
+    hex!("8974ef985b8dc87bbf14f16657f3bd30"),
+    hex!("89073a016fd5c618095291915c5912e1"),
+    hex!("86f16e07d569b7570af0031fb6c36af4"),
+    hex!("12d5be9063d155f9df791be6c35f1865"),
+    hex!("8962f4ccba3d7cb61f0a501d474f9906"),
+    hex!("efee4030b687d7ae3eb2d27673f65343"),
+    hex!("2464c007ea23451f778ed2bf0c017b7b"),
+    hex!("6dbe36b12f350e01689f6295d686b8b9"),
+    hex!("025cafd23d97228050066bf4128a4fa3"),
+    hex!("de0bb757deba77755678381d08a12bf1"),
+    hex!("1c542ed68bb6f818c4bee47703298d08"),
+    hex!("8178b368306f2ae1abd7e68583d67dad"),
+    hex!("cb4d956080d6f8bb617a2d51e5ac1296"),
+    hex!("06dc4c3c0f4d7c5f4784f3e865433730"),
+    hex!("7134fde96ed353c8d6ac24ec7efccaf9"),
+    hex!("1466a522411631d6c9c5c706558985c1"),
+    hex!("95e26e89a1c7b9df6a7c2c56223292b5"),
+    hex!("6555486abac007b7fadaf6e04f896239"),
+    hex!("472a1d76a9734964c42b92b36993ed49"),
+    hex!("cea176b441e44750555739f29e045e74"),
+    hex!("feb06e244cdbf8e8cb1d5fab49c6e583"),
+    hex!("499c5a7fc475c7ab9e888a9f8aeb5765"),
+    hex!("d3b05f45181c0b1b4a0354ffe3d95c9a"),
+    hex!("b768bcc1492f03e1819620dfa2e130d5"),
+    hex!("a38c13579500a7c7a90c80153ab22f4f"),
+    hex!("adb73d387c59a6ba8eff18a6b0320ca4"),
+    hex!("09c8e0bdd1ba73602063d89d11ff5210"),
+    hex!("ecab8f890f1f3d6ff564eebab034ebb3"),
+    hex!("9823e2e8911c5a0fb2ff6feb52be2c0f"),
+    hex!("4fa03dbaabcf99e71903f3177444bfaa"),
+    hex!("a4c9667d00998d262c47f6d2c79f43c2"),
+    hex!("b3e65ee26384d25bc7750b3f37e72883"),
+    hex!("7dea46d1a183f3070760eee0bf5fce3c"),
+    hex!("ecb2963c22a757569fe659635f4b0243"),
+    hex!("9ed968203144981e6e697db052910c27"),
+    hex!("4aa6ca6ebef942d8766065b2e590fd32"),
```

Comment:
Could you create an issue that we need to update these roots in around 1.5 years?

Reply:
Will do

Reply:
Done https://github.com/Snowfork/polkadot-ethereum/issues/158. I mentioned Q1 2022 as the deadline for this which should provide a few months' leeway before the node stops accepting headers